{"name": "test_simple_scenario", "status": "broken", "statusDetails": {"message": "pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"a simple step\". Line 3 in scenario \"Simple Scenario\" in the feature \"C:\\Users\\neete\\Downloads\\MinimalProject\\features\\simple.feature\"", "trace": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"a simple step\". Line 3 in scenario \"Simple Scenario\" in the feature \"C:\\Users\\neete\\Downloads\\MinimalProject\\features\\simple.feature\"\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "description": "C:\\Users\\neete\\Downloads\\MinimalProject\\features\\simple.feature: Simple Scenario", "start": 1743507591581, "stop": 1743507591582, "uuid": "7c5d269c-9011-49cb-a2c5-0088f74f3f65", "historyId": "e3c1535c6eddef9b2d529a85ce9dacda", "testCaseId": "e3c1535c6eddef9b2d529a85ce9dacda", "fullName": "tests.test_simple#test_simple_scenario", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_simple"}, {"name": "host", "value": "user"}, {"name": "thread", "value": "5164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_simple"}]}